src/main/java/codeu/controller/ActivityFeedServlet.java

```diff
@@ -0,0 +1,21 @@
+package codeu.controller;
+
+import java.io.IOException;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**import codeu.model.data.ActivityFeed;*/
```

Comment:
don't leave commented out code in commits! If you don't need this code, delete it :)